fossa/info/action.yml

```diff
@@ -2,6 +2,12 @@ name: fossa info
 
 description: Provides context info required for other FOSSA composite actions.
 
+inputs:
+  github-token:
+    description: |
+      The GitHub token to use for API calls.
+    required: true
+
```

Comment:
Am I right when saying, that this is a breaking change, when introduced?
I think we should therefore coordinate the merge of this and the subsequent PRs to avoid confusion amongst teams.

Reply:
Indeed, yes. I will create subsequent PRs.

Reply:
No breaking regarding that part.